### PR TITLE
Illinois

### PR DESCRIPTION
**Pull request by Andy-Developer94**
dont't sit at home, and go to the war

### CODE REVIEW

**Review by baryshev-ilja — COMMENTED** [review]
I kill you

**Comment by baryshev-ilja** [comment]
just do it

**Comment by baryshev-ilja** [comment]
............................

File: README.md [hunk 1]
```diff
@@ -1,3 +1,3 @@
 # Cat-energy
 Интернет-магазин кормов для домашних питомцев
-ramil is in the ass
```
Comment (Andy-Developer94) [inline README.md]: /////////

Comment (Andy-Developer94) [reply README.md]: sucker

**Comment by baryshev-ilja** [comment]
fawdfadf

**Comment by baryshev-ilja** [comment]
принимаю

**Pull request merged**